android/app/src/main/java/com/reactnativenavigation/views/SideMenu.java

```diff
@@ -76,11 +76,23 @@ public void setEnabled(boolean enabled, Side side) {
     }
 
     public void openDrawer(Side side) {
-        openDrawer(side.gravity);
+        try{
+            openDrawer(Gravity.LEFT);
```

Comment:
Is there no other way to determine the menu we should open?

Reply:
@guyca  I could not find any other way. this looked like the easiest way. Do we have any other work around?